### PR TITLE
Create firefox pref dir before use it

### PR DESCRIPTION
**Pull request by n1mh**
With brand, new workstations, `web_browser` policy uses `/etc/firefox/pref` folder without checking if it exists. This code check for this directory and creates if it doesn't exists.

### CODE REVIEW

File: providers/printers.rb [hunk 1]
```diff
@@ -79,8 +79,14 @@ def delete_printer(prt_name)
       printers_list.each do |printer|
         Chef::Log.info("Processing printer: #{printer.name}")
 
+        # HP printers are bad labeled in database
```
Comment (alfem) [inline providers/printers.rb]: This change has nothing to do with the subject of this PR.

Comment (n1mh) [inline providers/printers.rb]: I messed the branch up. I think I've already fix it by rolling it back.

Comment (alfem) [inline providers/printers.rb]: I still see them

Comment (n1mh) [inline providers/printers.rb]: changed. They are gone for good.

**Pull request merged**